rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -4676,6 +4676,13 @@ extern "C" rmw_ret_t rmw_take_response(
     info->reqtime.erase(seq);
   }
 #endif
+  TRACETOOLS_TRACEPOINT(
```

Comment:
can we also call `rmw_get_gid_for_client` to get client identification and add it to the tracepoint information?

Reply:
see https://github.com/ros2/ros2_tracing/pull/145#discussion_r1876858752